src/absil/ilwrite.fs

```diff
@@ -4116,7 +4129,6 @@ let writeBinaryAndReportMappings (outfile, ilg, pdbfile: string option, signer:
               match p with 
               | None -> () 
               | Some pExpected -> 
-                  os.Flush(); 
```

Comment:
`os.BaseStream` is a `MemoryStream` now instead of a `FileStream` so `Flush` is useless. 
[Because any data written to a MemoryStream object is written into RAM, this method is redundant](https://msdn.microsoft.com/en-us/library/system.io.memorystream.flush(v=vs.110).aspx)
I added `os.Flush` after the `gen` call

But `os` is a BinaryStream so maybe we need to leave that or add `AutoFlush = true` ( see [AutoFlush](https://msdn.microsoft.com/en-us/library/system.io.streamwriter.autoflush%28v=vs.110%29.aspx?f=255&MSPPError=-2147217396) ). I'll readd this `os.Flush`, less diff. Good catch :smile: